base/bootstrap.go

```diff
@@ -42,8 +42,12 @@ type BootstrapConnection interface {
 	// TouchMetadataDocument sets the specified property in a bootstrap metadata document for a given bucket and key.  Used to
 	// trigger CAS update on the document, to block any racing updates. Does not retry on CAS failure.
 	TouchMetadataDocument(ctx context.Context, bucket, key string, property string, value string, cas uint64) (casOut uint64, err error)
-	// KeyExists checks whether the specified key exists
+	// KeyExists checks whether the specified key exists in the bucket's default collection
```

Comment:
should we say metadata collection here and below for when we migrate this to a separate collection?

Right now default collection = metadata collection but it may not always.

Reply:
I intentionally left this comment as default collection because the bootstrap doesn't currently have the concept of a metadata store, and will require some refactoring when we want to add system collection support.